### DISCUSSION TITLE
Pricing and notification volume

### DISCUSSION THREAD

**Discussion by RuslanHryn** [magicbell/community#217] [Questions] [answered]
I would like to clarify the pricing calculation to understand better how notification volume is calculated.
For example, 

- If use only the app inbox, and send a notification to 5 users in a single request. Will it count as 5 notifications or 1 notification in pricing?
- if use only the app inbox, and send a notification to a topic in a single request, and the topic has 100 users, Will it count as 100 notifications or 1 notification in pricing?

**Comment by unamashana** [comment, marked as answer]
Hey @RuslanHryn we count notifications per user. So if a broadcast ends up with five recipients, that's five notifications. Whether that happens from listing users or via topics, it's the number of final recipients that matters.
